### PR TITLE
Removal of the word "top"

### PR DESCRIPTION
**Pull request by Valdnet**

### CODE REVIEW

**Review by AndyScherzinger — CHANGES_REQUESTED** [review]
Reading the diff: the "top" seems right

**Comment by Valdnet** [comment]
The disk usage space is not at the top of the personal page. 

obraz

Unless it's about the location on the menu, that is:

obraz

or where the user menu is located:

obraz

I also removed "the". My oversight.

If I'm wrong, correct me 😉.

File: user_manual/files/quota.rst [hunk 1]
```diff
@@ -3,7 +3,7 @@ Storage quota
 =============
 
 Your Nextcloud admin has the option to set a storage quota on users. Look at 
-the top of your Personal page to see what your quota is, and how much you have 
+the of your Personal page to see what your quota is, and how much you have
```
Comment (AndyScherzinger) [inline user_manual/files/quota.rst]: @Valdnet see here, the sentence is due to your change:

```
Look at the of your Personal page to see what your quota is, and how much you have used.
```

Which is not a proper sentence.

Did you forget to push your second change maybe?

File: user_manual/files/quota.rst [hunk 2]
```diff
@@ -2,9 +2,8 @@
 Storage quota
 =============
 
-Your Nextcloud admin has the option to set a storage quota on users. Look at 
-the top of your Personal page to see what your quota is, and how much you have 
-used.
+Your Nextcloud admin has the option to set a storage quota on users. Look at
+of your Personal page to see what your quota is, and how much you have used.
```
Comment (AndyScherzinger) [inline user_manual/files/quota.rst:6]: @Valdnet I'd say you would at least have to also remove "of" else the sentence is incorrect/incomplete, no?

**Review by AndyScherzinger — CHANGES_REQUESTED** [review]

**Comment by Valdnet** [comment]
You're right. One more change 😉.

**Review by AndyScherzinger — APPROVED** [review]

**Pull request merged**